src/database/index.tsx

```diff
@@ -44,6 +44,8 @@ const usersCollectionId = "users";
 // TODO (#521): when the datasource stabilizes, change this to the real collection
 const referenceFacilitiesCollectionId = "reference_facilities_test";
 const referenceFacilitiesCovidCasesCollectionId = "covidCases";
+// TODO (#559): when we switch to using the real data update this property for the scenario
```

Comment:
not that it hugely matters but i would tag this to #521, as in the comment above